Cooper/ClientApp/src.v2/app/components/user-info/user-info.component.html

```diff
@@ -14,4 +14,8 @@ <h5>E-mail:</h5>
     <h5>Bio:</h5>
     <p class="card-field">{{profile.description}}</p>
   </mat-card-content>
+
+  <div (click)="(isSubscription) ? onUnsubscribeButtonClicked() : onSubscribeButtonClicked()">
+    <button mat-raised-button class="my-button">{{(isSubscription)? 'Unsubscribe' : 'Subscribe'}}</button>
+  </div>
```

Comment:
it is better to make two separate buttons to make it more clear and easier to maintain